docs/kmesh_support.md

```diff
@@ -1,32 +1,16 @@
-### Native support all feature
+### Kmesh-capable OS
 
-We can directly compile and run all features of Kmesh in the following OS.
+Kmesh requires kernel eBPF functionality and sizeable eBPF Instruction Sets, so Kmesh can only run on operating systems above a certain version.
 
-| OS Version      | Kernel Version | Release Path                                |
-| :-------------: | :------------: | :-----------------------------------------: |
-| openEuler-23.03 |     6.1.19     | https://repo.openeuler.org/openEuler-23.03/ |
+Kmesh has two different modes, `Kernel-Native Mode` and `Duel-Engine Mode`. Although there is no difference between the two modes in terms of the os kernel version required, we still described separately in the table below.
 
-### Kmesh enhance kernel support all feature
+| Mode | OS Version | Kernel Version | Release Path |
+| :-------------: | :-------------: | :-------------: | :-------------: |
+| Kernel-Native Mode | openEuler-23.03 | 6.1.19 | https://repo.openeuler.org/openEuler-23.03/ |
+|                    | openEuler-22.03-LTS-SP1 |     5.10.0     | https://repo.openeuler.org/openEuler-22.03-LTS-SP1/ |
+|                    | Ubuntu Server 22.04.3 LTS |     5.15.0   |         https://releases.ubuntu.com/jammy/          |
+| Duel-Engine Mode | openEuler-23.03 | 6.1.19 | https://repo.openeuler.org/openEuler-23.03/ |
+|                    | openEuler-22.03-LTS-SP1 |     5.10.0     | https://repo.openeuler.org/openEuler-22.03-LTS-SP1/ |
+|                    | Ubuntu Server 22.04.3 LTS |     5.15.0   |         https://releases.ubuntu.com/jammy/          |
```

Comment:
This is still  consuding, can you remove the OS version for dual engine

And i think kernel native mode, this is also not quire right